plutus-core/plutus-ir/test/transform/inline/letOverAppMulti.golden

```diff
@@ -17,6 +17,6 @@
       )
       (lam x (fun (con integer) (con integer)) idFun)
     )
-    (con integer 6)
+    [ [ k idFun ] (con integer 6) ]
```

Comment:
regression?

Reply:
This seems indeed a regression. I'm not sure why `k` is not inlined here. `k idFun` reduces to `idFun`, which is clearly a win.